qframe.go

```diff
@@ -397,6 +397,40 @@ func (qf QFrame) ColumnNames() []string {
 	return result
 }
 
+// Columns returns all ordered underlying column.Column
+
+// Time complexity O(n) where n = number of columns.
+func (qf QFrame) Columns() []column.Column {
```

Comment:
Does exposing `column.Column` like this even work when the caller is outside the qframe directory.

I placed the `Column` interface in an internal package right now to avoid having people rely on the content of it. In the future, once it's somewhat stable, it would probably make sense to make it public though so that users can pass in any column type they please to the constructor, including their own implementations backed by whatever data.

Perhaps we can make it  public now already without making too many promises on API stability though...

Reply:
It does indeed work however looking at this again I feel it's not really needed and will revert this.